src/viewer.ts

```diff
@@ -295,6 +295,7 @@ class TrackableViewerState extends CompoundTrackable {
     this.add("layers", viewer.layerSpecification);
     this.add("showAxisLines", viewer.showAxisLines);
     this.add("wireFrame", viewer.wireFrame);
+    this.add("enableAdaptiveDownsampling", viewer.enableAdaptiveDownsampling);
```

Comment:
Add corresponding option to Python viewer_state.py

Reply:
Thanks @jbms ! I added it in https://github.com/google/neuroglancer/pull/591/commits/60ca39c0389d02e623c937d39c05350d7d1a026e

It looks like the wireframe setting was also missing, so I added it too. But let me know if that's not right https://github.com/google/neuroglancer/pull/591/commits/c9ef6d925deff20bf99dad8e6973be07aa53900e